task/fips-operator-bundle-check/0.1/fips-operator-bundle-check.yaml

```diff
@@ -0,0 +1,136 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "konflux"
+  name: fips-operator-bundle-check
+spec:
+  description: >-
+    Checks operator bundle image builds for FIPS compliance using the check-payload tool.
+  params:
+    - name: image-digest
+      description: Image digest to scan.
+    - name: image-url
+      description: Image URL.
+  results:
+    - name: TEST_OUTPUT
+      description: Tekton task test output.
+      value: $(steps.fips-operator-check-step-action.results.TEST_OUTPUT)
+    - name: IMAGES_PROCESSED
+      description: Images processed in the task.
+      value: $(steps.fips-operator-check-step-action.results.IMAGES_PROCESSED)
+  steps:
+    - name: get-unique-related-images
+      image: quay.io/redhat-appstudio/konflux-test:v1.4.9@sha256:eee855e60b437d9a55a30e63f2eb7f95d9fd6d3b111c32cac8730c9b7a071394
+      computeResources:
+        limits:
+          memory: 512Mi
+          cpu: 200m
+        requests:
+          memory: 256Mi
+          cpu: 100m
+      env:
+        - name: IMAGE_URL
+          value: $(params.image-url)
+        - name: IMAGE_DIGEST
+          value: $(params.image-digest)
+      results:
+        - name: unique_related_images
+          description: Unique relatedImages from input operator bundle image
+        - name: images_processed
+          description: Images processed in the task.
+      securityContext:
+        capabilities:
+          add:
+            - SETFCAP
+      script: |
+        #!/usr/bin/env bash
+        set -euo pipefail
+        # shellcheck source=/dev/null
+        . /utils.sh
+
+        imagewithouttag=$(echo -n "${IMAGE_URL}" | sed "s/\(.*\):.*/\1/")
```

Comment:
It is consistent with other build-pipeline tasks like clair-scan, clamav-scan. I think we are certain it will not have the digest in it

Reply:
because the IMAGE_URL is effectively the target tag which the image will be pushed to... it'll just be a big bug to resolve across the board if we run into an issue.

Reply:
We can maybe add a task to our board to unify functions for parsing image references across at least the testing/scanning tasks?